app/serializers/user_serializer.rb

```diff
@@ -44,4 +50,18 @@ class UserSerializer
       CIP_TYPES[:none]
     end
   end
+
+  attributes :induction_programme_choice do |user|
+    if user.participant?
+      school_cohort = SchoolCohort.find_by(school: user.school)
+      case school_cohort.induction_programme_choice
+      when "full_induction_programme"
+        PROGRAMME_TYPES[:full_induction_programme]
+      when "core_induction_programme"
+        PROGRAMME_TYPES[:core_induction_programme]
+      when "not_yet_known"
+        PROGRAMME_TYPES[:not_yet_known]
+      end
+    end
+  end
```

Comment:
This is a noop, right?

Reply:
Is it? I think the `case when` statement will return some string (symbol?), then if statement will return what the case statement returns, and the block will return what if returns. I hope. We could probably add an explicit `return`, but I am not sure rubocop would be happy with it.

Reply:
My point was this was the same as returning `school_cohort.induction_programme_choice`. Sorry that wasn't clear

Reply:
The Case specifically is an identity mapping

Reply:
Oh, right. Well, not quite, I think this case will return some `nil`s when `induction_programme_choice` doesn't, but I agree we can probably just return `school_cohort .induction_programme_choice` here.

Although in principle that means changing the values of `induction_programme_choice` enum will be harder, I don't think it's likely to ever happen.

Reply:
I think we would add this back in if we changed it, but it's just odd to have it here doing nothing